sonic_platform_base/sonic_xcvr/api/xcvr_api.py

```diff
@@ -637,3 +645,35 @@ def get_error_description(self):
         """
         raise NotImplementedError
 
+    def get_overall_offset(self, page, offset, size, wire_addr=None, flat=False):
+        """
+        Retrieves the overall offset of the given page, offset and size
```

Comment:
@Junchao-Mellanox you mean the absolute offset when seen EEPROM has a single flat memory?

Reply:
Yes